apis/projectcontour/v1/httpproxy.go

```diff
@@ -327,6 +327,24 @@ type VirtualHost struct {
 	// +optional
 	JWTProviders []JWTProvider `json:"jwtProviders,omitempty"`
 
+	// The policy for managing request headers during proxying.
+	// Headers are appended to requests in the following order,
+	// weighted cluster level headers,
```

Comment:
nit: let's maybe not use "weighted cluster" just since that is an explicitly Envoy rather than Contour-specific term, we can use Service instead I think

Reply:
Updated